models/aeif_cond_alpha.nestml

```diff
@@ -86,11 +86,11 @@ neuron aeif_cond_alpha_neuron:
 
     # Impulse to add to DG_EXC on spike arrival to evoke unit-amplitude
     # conductance excursion.
-    PSConInit_E nS/ms = 1.0 nS * e / tau_syn_ex
+    PSConInit_E nS/ms = nS * e / tau_syn_ex
```

Comment:
it's a detail, but I would feel more comfortable if units were put at the end, as a general rule (it is more intuitive)

This remark applies throughout the PR